homeassistant/components/zwave_me/sensor.py

```diff
@@ -0,0 +1,114 @@
+"""Representation of a sensorMultilevel."""
+from __future__ import annotations
+
+from zwave_me_ws import ZWaveMeData
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_ILLUMINANCE,
+    DEVICE_CLASS_POWER,
+    DEVICE_CLASS_SIGNAL_STRENGTH,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_VOLTAGE,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ZWaveMeEntity
+from .const import DOMAIN
+
+SENSORS_MAP: dict[str, SensorEntityDescription] = {
+    "meterElectric_watt": SensorEntityDescription(
+        key="meterElectric_watt",
+        device_class=DEVICE_CLASS_POWER,
+        native_unit_of_measurement="W",
```

Comment:
You should import and use the unit constants when they are available.
For example:
`from homeassistant.const import POWER_WATT`
```suggestion
        native_unit_of_measurement=POWER_WATT,
```

Reply:
Fixed with a new commit in this PR